### PR TITLE
Update Documentation

### LINKED ISSUE #1: Add documentation
Add proper documentation

### PR DESCRIPTION
**Pull request by huf0813**
@gifff @adityapp @zainfikrih 
Hello, i want to help updating the documentation. Thanks!

### CODE REVIEW

File: .idea/jarRepositories.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      <option name="name" value="BintrayJCenter" />
+      <option name="url" value="https://jcenter.bintray.com/" />
+    </remote-repository>
+    <remote-repository>
+      <option name="id" value="Google" />
+      <option name="name" value="Google" />
+      <option name="url" value="https://dl.google.com/dl/android/maven2/" />
+    </remote-repository>
+  </component>
+</project>
```
Comment (gifff) [inline .idea/jarRepositories.xml]: I wonder what this file and compiler.xml are used for.
Do you know?

**Review by gifff — APPROVED** [review]
I add some notes in the README
Please also check it again

**Comment by ekaiqbalv** [comment]
I think you only need to commit README.md file. And maybe it would be better if you add an explanation that this library has provided image provider options that can be used by users.

**Comment by huf0813** [comment]
> I think you only need to commit README.md file. And maybe it would be better if you add an explanation that this library has provided image provider options that can be used by users.

Hi, i have some changes. Please consider to check my latest changes. Thanks. @ekaiqbalv

**Review by ekaiqbalv — APPROVED** [review]

**Pull request merged**